### PR TITLE
Get number of months between two dates

### PR DESCRIPTION
**Pull request by kwaimind**
Adding a useful date method that gets the number of months between two dates. It's a long one so I'm happy to uglify it a bit more if needed.

### CODE REVIEW

File: collections/date-time/calculate-the-number-of-months-between-two-dates.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+~~~ javascript
+const monthDiff = (startDate, endDate)  => (endDate.getFullYear() - startDate.getFullYear()) * 12 - startDate.getMonth() + endDate.getMonth() <= 0 ? 0 : (endDate.getFullYear() - startDate.getFullYear()) * 12 - startDate.getMonth() + endDate.getMonth()
```
Comment (phuocng) [inline collections/date-time/calculate-the-number-of-months-between-two-dates.md]: Should it be
`Math.max(0, (endDate.getFullYear() - startDate.getFullYear()) * 12 - startDate.getMonth() + endDate.getMonth())`?

Comment (kwaimind) [reply collections/date-time/calculate-the-number-of-months-between-two-dates.md]: Yeah nice that's much cleaner. I'll make a new commit.

**Comment by phuocng** [comment]
Thank you, @kwaimind. 
The snippet is now available here https://1loc.dev/#calculate-the-number-of-months-between-two-dates

**Comment by kwaimind** [comment]
Amazing. Thanks @phuoc-ng 🎈

**Pull request merged**